src/store/FlowerStore.java

```diff
@@ -0,0 +1,77 @@
+package store;
+
+import java.util.ArrayList;
+
+public class FlowerStore {
+    private ArrayList<String> firstBouquet;
```

Comment:
program to interface not to implementation
but in this case you should use array